api/src/main/java/org/openmrs/api/impl/PatientSetServiceImpl.java

```diff
@@ -249,7 +249,9 @@ public Cohort getPatientsHavingDrugOrder(Collection<Integer> patientIds, Collect
 		} else {
 			if (groupMethod == GroupMethod.NONE) {
 				// Patients taking none of the specified drugs
-				
+				if (patientIds == null) {
```

Comment:
I wonder what you're coding against, I see this check [here](https://github.com/ldf92/openmrs-core/blob/PatientSetServiceTest/api/src/main/java/org/openmrs/api/impl/PatientSetServiceImpl.java#L239) already